.circleci/config.yml

```diff
@@ -95,6 +95,31 @@ jobs:
           name: Slither
           command: | 
               slither . --filter-paths "node_modules" --disable-color 
+              
+  gasCompare:
+    docker:
+      - image: 'cimg/node:16.5'
+    resource_class: large
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Run GasCompare
+          command: |
+              if [ "$CIRCLE_BRANCH" == 'master' ]; then
+              exit 0
+              fi
+              CI=true npm run test
+              mkdir temp
+              mv ./gasReporterOutput.json ./temp/gasReporterOutput_Current.json
```

Comment:
circle CI image has a /tmp I think. Please check.

Reply:
ok

Reply:
Fixed it